gen-windows.bat

```diff
@@ -1,26 +0,0 @@
-@echo off
```

Comment:
Don't delete it, back this file and just update command with news version please